### PR TITLE
(Fix) full width patient chart widgets meta

### PR DESCRIPTION
**Pull request by pirupius**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
Currently all widgets on the patient chart are restricted to cover a `max-width: 60rem` which works well for the most part. with small widgets but this leaves a lot of redundant space and squeezed widgets when the widgets have many columns or a lot of data to display. 

With this implementation this restriction was extracted to another style class that is added conditionally if the `layoutMode` key is set in the `meta` and/or `routes.json`

```
.widthContained {
  max-width: 60rem;
}
```

## Screenshots

When not set or `layoutMode:  'contained'` 

Screenshot 2023-08-14 at 13 33 24

Screenshot 2023-08-14 at 13 37 10

When `layoutMode:  'anchored'` 

Screenshot 2023-08-14 at 13 33 41

Screenshot 2023-08-14 at 13 36 43

### CODE REVIEW

File: packages/esm-patient-chart-app/src/patient-chart/chart-review/chart-review.component.tsx [hunk 1]
```diff
@@ -27,16 +27,13 @@ interface ChartReviewProps {
   patientUuid: string;
   patient: fhir.Patient;
   view: string;
+  fullWidthWidgets?: (isWidgetFullWidth: boolean) => void;
```
Comment (pirupius) [inline packages/esm-patient-chart-app/src/patient-chart/chart-review/chart-review.component.tsx:30]: The naming might not be the best so am open to ideas for better options

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx [hunk 2]
```diff
@@ -71,8 +72,8 @@ const PatientChart: React.FC = () => {
                 <ExtensionSlot name="patient-info-slot" state={state} />
               </aside>
               <div className={styles.grid}>
-                <div className={styles.chartReview}>
-                  <ChartReview {...state} view={view} />
+                <div className={`${styles.chartReview} ${!isWidgetFullWidth ? styles.widthContained : ''}`}>
```
Comment (pirupius) [inline packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx:75]: Conditionally add style based on chart meta

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss [hunk 3]
```diff
@@ -19,11 +19,14 @@ $actionPanelExpandedOffset: $actionNavOffset+$actionPanelOffset;
...
   height: 90%;
   width: 100%;
-  max-width: 60rem;
   margin: 0 auto;
   padding-bottom: spacing.$spacing-10;
 }
 
+.widthContained {
+  max-width: 60rem;
+}
```
Comment (pirupius) [inline packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:28]: Extracted this to it's own class for easier managing

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx [hunk 4]
```diff
@@ -71,8 +73,8 @@ const PatientChart: React.FC = () => {
                 <ExtensionSlot name="patient-info-slot" state={state} />
               </aside>
               <div className={styles.grid}>
-                <div className={styles.chartReview}>
-                  <ChartReview {...state} view={view} />
+                <div className={`${styles.chartReview} ${layoutMode == 'contained' ? styles.widthContained : ''}`}>
+                  <ChartReview {...state} view={view} dashboardLayoutMode={setLayoutMode} />
```
Comment (samuelmale) [inline packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx:77]:
```suggestion
                  <ChartReview {...state} view={view} setDashboardLayoutMode={setLayoutMode} />
```

**Review by samuelmale — APPROVED** [review]
Nice work @pirupius. Just a few minor suggestions

**Pull request merged**